src/third-party/PhishingController.ts

```diff
@@ -135,18 +137,26 @@ export class PhishingController extends BaseController<
       return;
     }
 
-    const phishingOpts = await this.queryConfig(this.configUrl);
+    const [phishingOpts, pfBlacklist] = await Promise.all([
+      await this.queryConfig<EthPhishingResponse>(this.configUrlMetaMask),
```

Comment:
Ideally we'd keep using one list if the other fails, rather than use our fallback. Right now if one failed, both results would get thrown out.

Perhaps `queryConfig` could be updated to catch network errors and log them without throwing?